dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/InstallKernelMap.java

```diff
@@ -708,4 +774,70 @@ private Integer uninstall() {
         }
         return OK;
     }
-}
+
+    private static void getShortNameFromManifest(File esa, Map<String, String> shortNameMap) throws IOException {
```

Comment:
Change method name to be more descriptive, since it is currently named "get..." but does not return anything.  Perhaps call it "populateShortNameFromManifest"